docs-conceptual/azps-4.5.0/manage-azure-resources-invoke-azrestmethod.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Manage Azure resources with Invoke-AzRestMethod
+description: How to use Azure PowerShell to manage resources with the Invoke-AzRestMethod cmdlet.
+ms.devlang: powershell
+ms.topic: conceptual
+ms.date: 08/17/2020
+---
+
+# Manage Azure resources with Invoke-AzRestMethod
+
+[Invoke-AzRestMethod](/powershell/module/az.accounts/invoke-azrestmethod) is an Azure PowerShell
+cmdlet that was introduced in Az PowerShell module version 4.4.0. It allows you to make custom HTTP
+requests to the Azure Resource Management (ARM) endpoint using the Az context.
+
+This cmdlet is useful when you want to manage Azure services for features that aren't yet available
+in the Az PowerShell module.
+
+## How to use Invoke-AzRestMethod
+
+As an example, you can allow access to Azure Container Registry (ACR) only for specific networks or
+deny public access. This feature isn't available yet in the [Az.ContainerRegistry PowerShell module](/powershell/module/Az.ContainerRegistry/).
```

Comment:
We sill eventually add this in the ACR module so we should have a link that includes the ACR version without this capability.